src/tests/networking_tests.rs

```diff
@@ -8,28 +8,30 @@ pub mod net_tests {
     use std::sync::{Arc, Barrier};
 
     pub fn net_tests() {
-        ut_lind_net_bind();
```

Comment:
Was this due to 'cargo fmt' or did you do this for some reason?

I can see the writev test was removed.  Why?

Reply:
If you've fixed these, please "resolve conversation"  If all are done and you're ready for another review, please add a comment or similar to say so.